css_tasks/task11/index.html

```diff
@@ -0,0 +1,896 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Task 11</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Kaushan+Script|Montserrat:400,700|Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <!-- HOME -->
+    <section class="home">
+      <div class="home__background">
+        <picture>
+          <source media="(max-width:678px)" srcset="assets/layer-15-min.jpg" />
+          <source
+            media="(min-width:679px) and (max-width:1024px)"
+            srcset="assets/layer-15@2x-min.jpg"
+          />
+          <img
+            src="assets/layer-15@3x-min.jpg"
+            alt="Eagle"
+            class="home-background__image"
+          />
+        </picture>
+      </div>
+      <header class="header container">
+        <span><a href="#" class="header__logo">MoGo</a></span>
+        <nav class="header__nav">
+          <ul>
+            <li class="header-nav__item"><a href="#">About</a></li>
+            <li class="header-nav__item"><a href="#">Service</a></li>
+            <li class="header-nav__item"><a href="#">Work</a></li>
+            <li class="header-nav__item"><a href="#">Blog</a></li>
+            <li class="header-nav__item"><a href="#">Contact</a></li>
+            <li class="header-nav__item">
+              <span
+                ><img src="assets/shopping-cart-solid.svg" alt="shopping cart"
+              /></span>
+            </li>
+            <li class="header-nav__item">
+              <span><img src="assets/search-solid.svg" alt="search"/></span>
+            </li>
+          </ul>
+        </nav>
+        <span class="header-nav__mobile" role="button"
+          ><img src="assets/bars-solid.svg" alt="mobile menu"
+        /></span>
+      </header>
+      <div class="home__inner container">
+        <h1>Creative Template</h1>
+        <p>
+          Welcome <br />
```

Comment:
maybe it is possible to do with sizes and so on

Reply:
OK